README.md

```diff
@@ -1,58 +1,71 @@
 
 ![CastMate UI Profiles](docs/images/profile.png?raw=true)
 
-### Setup
+The possibilities are endless, so here are some ideas to get you started:
 
-1. Download the installer [here](https://www.github.com/LordTocs/CastMate/latest)
+-   Create a text-to-speech channel point redemption that's only available as long as the stream has less than some number of viewers.
+-   A "Stream Ending" profile which automatically pastes social media links to chat when your "End Stream" scene is active in OBS.
+-   Create a nearby enemy gunfire sound effect that's only available during specific games (so it can't be used during an ASMR stream, for example). Alternatively, make a jump scare sound effect cost triple during ASMR.
+-   Have a song request channel redemption that's only available during a Music stream segment.
+-   Create an alternate versions of a channel point redemption that costs more to redeem when the number of viewers is higher, to encourage engagement during slow streams and to reduce spam during active streams.
+-   Set up a channel point redemption that is only visible when there are exactly 69 viewers, which can only be redeemed once per stream or once per viewer.
+
+## Setup
+
+1. Download `CastMate-Setup.exe` from GitHub releases: https://www.github.com/LordTocs/CastMate/releases
 
 2. Run the installer
 
 3. Sign in with your twitch account, and optionally a bot account. ![CastMate Sign In](docs/images/sign-in.png?raw=true)
 
 4. See the [OBS Setup Guide](OBS.md) to configure OBS!
 
-## Download
-
-Download from GitHub releases: https://www.github.com/LordTocs/CastMate/releases
-
 ## Upcoming Features
 
-* Downloadable Plugins - Allow additional plugins to be installed to support new actions, triggers, state, and other features!
-* OBS Websocket Plugin 5.x - With OBS 28.0.0 the websocket plugin will be bundled automatically with OBS! Reducing our setup overhead!
-* Overlay Editor - Create overlays to show custom alerts and other visual effects. 
-* State Triggers - Run automations when CastMate state triggers.
-* BitButtons - Let viewers trigger automations via bits!
+-   Downloadable Plugins - Allow additional plugins to be installed to support new actions, triggers, state, and other features!
+-   OBS Websocket Plugin 5.x - With OBS 28.0.0 the websocket plugin will be bundled automatically with OBS! Reducing our setup overhead!
+-   Overlay Editor - Create overlays to show custom alerts and other visual effects.
+-   State Triggers - Run automations when CastMate state triggers.
+-   BitButtons - Let viewers trigger automations via bits!
 
 ## Contributing
 
 Pull Requests are on hold until a CLA can be worked out.
 
 ## Support
 
-- [LordTocs' Twitch](https://www.twitch.tv/lordtocs)
-- [LordTocs' YouTube](https://www.youtube.com/channel/UCe4uXUoF5MkKvhgy514FCuA)
-
+Feedback and bug reports are greatly appreciated! Please don't hesitate to reach out through Twitch, Discord, or GitHub issues.
```

Comment:
An invite link to discord would be great. I generated one yesterday https://discord.gg/txt4DUzYJM